packages/client/src/Accounts.tsx

```diff
@@ -40,37 +112,84 @@ export default function Accounts(_props: RouteComponentProps) {
 
   return (
     <div>
-      {accounts.map(account => (
-        <section key={account.id}>
-          <header>
-            <h3>{account.email}</h3>
-            {account.loggedIn ? (
-              <Link to={`/accounts/${account.id}/dashboard`}>
-                View Conversations
-              </Link>
-            ) : (
-              <button
-                onClick={() =>
-                  authenticate({ variables: { id: account.id } }).catch(
-                    setError
-                  )
-                }
-              >
-                Log In
-              </button>
-            )}
-          </header>
-        </section>
-      ))}
-      <form onSubmit={onSubmit}>
-        <input
-          type="text"
-          name="email"
-          onChange={e => setEmailValue(e.target.value)}
-          value={emailValue}
-        />
-        <button type="submit">Add Account</button>
-      </form>
+      <div className={classes.appBarSpacer} />
+      <AppBar>
+        <Toolbar>
+          <Typography variant="h6">Poodle</Typography>
+        </Toolbar>
+      </AppBar>
+      <div className={classes.root}>
+        {accounts.map(account => (
+          <section key={account.id}>
+            <header>
+              <Card className={classes.card}>
+                <CardHeader title={account.email} />
+                <CardActions>
+                  {account.loggedIn ? (
+                    <>
+                      <Link
+                        to={`/accounts/${account.id}/dashboard`}
+                        className={classes.link}
+                      >
+                        <Button>View Conversations</Button>
+                      </Link>
```

Comment:
This looks like it works; but the idiomatic way to combine a `Button` with a custom link component is to put the link in the button's `component` property. I think you want something like this:

```typescript
<Button component={Link} to={`/accounts/${account.id}/dashboard`}>View Conversations</Button>
```

By default the `Button` component will render a native HTML `<a>` or `<button>` tag - but if you specify `component={Link}` it will render a `<Link>` element instead, and will pass the `to` prop forward to that `Link`.